### DISCUSSION TITLE
NVENC preset bug?

### DISCUSSION THREAD

**Discussion by Mishasama** [Xaymar/obs-stream-effects#1009] [General]
OBS 29 beta 3
SFX 12.0b164
Preset LLHQ

It will eat all my NVENC power, but LLHP not. And with the same settings on older OBS 27 & SFX versions, it won't.
Does anyone have the same problem?

**Comment by Xaymar** [comment]
Only the p1-p7 presets are supported on recent drivers

**Reply by Mishasama** [reply]
> Only the p1-p7 presets are supported on recent drivers

Then why not delete these options in SFX?

After upgrade to OBS 29 stable, SFX was gone. Do I need to reinstall SFX? Or wait SFX release an update?

**Reply by Xaymar** [reply]
StreamFX exposes whatever FFmpeg exposes. It is your own duty to be informed about what is and isn't supported - StreamFX is for power users after all.
